spec/factories/restaurant_factory.rb

```diff
@@ -0,0 +1,31 @@
+# == Schema Information
+#
+# Table name: restaurants
+#
+#  id                    :uuid             not null, primary key
+#  accept10bis           :boolean          default(FALSE), not null
+#  address               :string
+#  cuisine               :string
+#  latitude              :float
+#  longitude             :float
+#  max_delivery_time_min :integer
+#  name                  :string           not null
+#  rating                :float            default(0.0)
+#  created_at            :datetime         not null
+#  updated_at            :datetime         not null
+#
+
+require 'faker'
+
+FactoryBot.define do
+  factory :restaurant, traits: [:name, :cuisine] do
+    trait :name do
+      name { Faker::Restaurant.name }
+    end
+    trait :cuisine do
+      cuisine { Restaurant::E_CUISINES.sample }
```

Comment:
traits are used to show different states of the object. You should usually apply all the properties as default (name, cuisine) and have traits override the ones you don't want.